packages/cubejs-dremio-driver/README.md

```diff
@@ -9,11 +9,24 @@
 
 Pure Javascript Dremio driver.
 
+## Dremio Cloud
+
+To use this driver with [Dremio Cloud](https://docs.dremio.com/cloud/reference/api/), use the following setup:
+
+| Environment Variable | Value                                              |
+| -------------------- | -------------------------------------------------- |
+| CUBEJS_DB_TYPE       | dremio                                             |
+| CUBEJS_DB_URL        | https://api.dremio.cloud/v0/projects/${PROJECT_ID} |
+| CUBEJS_DB_NAME       | ${DB_NAME}                                         |
+| CUBEJS_DB_PASS       | Bearer ${PERSONAL_ACCESS_TOKEN}                    |
```

Comment:
Let's not mess everything in one place and introduce another special env var for token. You can do it in https://github.com/cube-js/cube/blob/master/packages/cubejs-backend-shared/src/env.ts 
Have a look at Pinot / Startree Auth Token for example.
Something like this:
```js
  /** ***************************************************************
   * Dremio Driver                                                  *
   **************************************************************** */

  /**
   * Dremio cloud Auth Token
   */
  dremioAuthToken: ({
    dataSource,
  }: {
    dataSource: string,
  }) => (
    process.env[
      keyByDataSource('CUBEJS_DB_DREMIO_AUTH_TOKEN', dataSource)
    ]
  ),

```

Reply:
Good idea, change has been made to support this.